src/Relude/Bifoldable.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE CPP #-}
+
+{- |
+Copyright:  (c) 2018 astynax
+License:    MIT
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+
+This module reexports some 'Bifoldable' related stuff.
+-}
+
+module Relude.Bifoldable
+       (
+#if MIN_VERSION_base(4,10,0)
+         Bifoldable (..)
```

Comment:
You can reexport just `module Data.Bifoldable`. You've already specified what this module imports, so it's okay to shorten reexport clause